src/Microsoft.Extensions.CodeGeneration.Utils/DotNet/LibraryExporterExtensions.cs

```diff
@@ -21,29 +21,31 @@ public static string GetResolvedPathForDependency(this ILibraryExporter _library
             {
                 throw new ArgumentNullException(nameof(library));
             }
-
             var exports = _libraryExporter.GetAllExports();
+            if(exports == null)
+            {
+                return string.Empty;
+            }
             var assets = exports
-                .SelectMany(export => export.RuntimeAssemblyGroups.GetDefaultGroup().Assets)
+                .SelectMany(export => export.RuntimeAssemblyGroups?.GetDefaultGroup()?.Assets)
                 .Where(asset => asset.Name == library.Identity.Name);
-
-            if (assets.Any())
+            if (assets != null && assets.Any())
```

Comment:
assets can't be null, you just did a `SelectMany` on it, would have null ref'd, same below.